docs/guides/index.md

```diff
@@ -22,6 +22,7 @@ These guides cover marimo's core concepts.
 | [Using your own editor](editor_features/watching.md)  | Edit notebooks in your own editor and stream changes back to the browser |
 | [Apps](apps.md)                                       | Running notebooks as apps                                  |
 | [Scripts](scripts.md)                                 | Running notebooks as scripts                               |
+| [Libraries](library.md)                               | Reusing notebook definitions                               |
```

Comment:
To be more specific:

```suggestion
| [Reusing functions and classes](library.md)                               | Importing functions and classes defined in marimo notebooks                             |
```

The navbar title in `mkdocs.yml` should also be `Reusing functions and classes`.